pagopa/io-app
# chore: [PE-562] Refactor bonus information screen to use new DS components

> [!note]
> This PR Depends on pagopa/io-services-metadata#771 and pagopa/io-dev-api-server#357

## Short description
This PR refactors the CGN Bonus information screen to proper use new DS components and UI.

## List of changes proposed in this pull request
- Changed CGN navigator to use navigation header
- Refactor on BonusInformationScreen to use DS components
- Extended RNavScreenWithLargeHeader to include a possible Hero cover image

## How to test
Using the dev-server on code from pagopa/io-dev-api-server#357 navigate to CGN initial activation screen to check the new UI.

Here the [Figma](https://www.figma.com/file/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?type=design&node-id=4403-98761&mode=design&t=uFMW3F4RlE3uLlU4-4) for reference.

| Experimental DS ✅  | Experimental DS ❌  |
|---|---|
| <img src="https://github.com/pagopa/io-app/assets/3959405/9212ed58-dc30-450d-b478-f870633b207e" height="700"/> | <img src="https://github.com/pagopa/io-app/assets/3959405/adc1cbcd-db68-4914-927b-21c468847f57" height="700"/> |
